happi/tests/test_cli.py

```diff
@@ -44,14 +44,14 @@ def trim_split_output(strings: str, delim: str = '\n'):
     """
     date_pattern = r"\[(\d{4})[-](0[1-9]|1[012])[-].*\]"
 
-    ok_substrs = [r"happi.item."]
+    omit_substrs = [r"happi.containers.", r"- DEBUG -"]
```

Comment:
I'm not sure I actually understand what this function is trying to accomplish, so it's also hard to understand the edits

Reply:
There was apparently some output that had datetime information, as well as debug prints.  This function removes those from the observed output so the tests aren't time-dependent.

The containers bit is to remove containers collected from entrypoints.  (eg pcdsdevices).  this is admittedly not very robust, since external containers aren't guaranteed to be in a submodule containing 'happi.containers.'  
(in happi add we list all the available containers, and if you run this test suite in psbuild it'll find all the pcdsdevices containers)